bundles/org.openhab.automation.pythonscripting/src/main/java/org/openhab/automation/pythonscripting/internal/PythonScriptEngine.java

```diff
@@ -121,7 +121,7 @@ public class PythonScriptEngine
             .targetTypeMapping(Value.class, Duration.class,
                     // picking two members to check as Duration has many common function names
                     v -> v.hasMember("total_seconds") && v.hasMember("total_seconds"),
-                    v -> Duration.ofNanos(v.invokeMember("total_seconds").asLong() * 10000000),
+                    v -> Duration.ofNanos(v.invokeMember("total_seconds").asLong() * 1000000000),
```

Comment:
```suggestion
                    v -> Duration.ofSeconds(v.invokeMember("total_seconds").asLong()),
```

Reply:
another commit which fix the nano seconds accuracy and make it clear why we need "Duration.ofNanos"